User-Help/Scoring

```diff
@@ -0,0 +1,22 @@
+
+# Question scoring and ordering
```

Comment:
Ordering applies to answers.  Questions shown in question lists are, by default, ordered by recency.  (Yeah you might sort them by score, but since it's not the default, let's focus on the bigger impact -- answers.)